exercises/ruby-1-b/ruby_1_b_example.rb

```diff
@@ -1,12 +1,24 @@
-VOWELS = %w{a e i o u}
-def word_with_most_vowels(sentence)
-  words = sentence.split(" ")
-  word_vowels = words.map do |word|
-    [
-      word,
-      word.downcase.chars.select{|l|VOWELS.include?(l)}.count
-    ]
+class Library
```

Comment:
My slight worry is that this exercise might be a little bit too simple. I'm not a Ruby programmer though. Maybe @iHiD has some thoughts?